### ISSUE TITLE
Mac result types are not exported

### ISSUE THREAD

**Issue by Pandawan** [hoodie/notify-rust#138]
Hey! I was trying to set the mac bundle identifier and encountered a small issue when trying to pattern match them.
Unfortunately, it seems that notify-rust does not export the necessary types to match e.g. `mac_notification_sys`'s error types. (The examples simply `unwrap`, thus avoiding the issue).

I think I can simply make my project depend on `mac_notification_sys` so I can just use the types directly from there, but perhaps it'd be cool to have those be available somehow? There has to be some way to make this more ergonomic...

Example:
```rust
if cfg!(target_os = "macos") {
    match notify_rust::set_application("com.apple.Terminal") {
        Ok(_) => { /* Do something*/ }
        Err(______) => { /* Handle a specific error */ }
        Err(err) => { /* Fallback for other errors */ }
    }
}
```

PS: Maybe I'm missing something? I see that there's an `extern crate` however, I'm unsure how to use it. I can't `use mac_notification_sys` without having it be part of the cargo project.

**Comment by hoodie** [comment]
oops :D

**Comment by hoodie** [comment]
@Pandawan wanna give `v4.5.8-pre.1` a try?

**Comment by hoodie** [comment]
I think https://github.com/hoodie/notify-rust/blob/feature/fix-mac-exports/examples/mac_app_id.rs should resemble your use-case right?

**Comment by hoodie** [comment]
I just released 4.5.8 - please reopen this if it's not sufficient

**Comment by Pandawan** [comment]
Yup, thanks for the quick fix!
